registry/default/example/interactive-hover-button-demo.tsx

```diff
@@ -1,9 +1,9 @@
-import InteractiveHoverButton from "@/registry/default/magicui/interactive-hover-button";
+import { InteractiveHoverButton } from "@/registry/default/magicui/interactive-hover-button";
 
 export default function InteractiveHoverButtonDemo() {
   return (
     <div className="relative justify-center">
```

Comment:
we can remove this surrounding div to make it even simpler

check out shiny button demo for example

Reply:
Yes we can.

Reply:
awesome